notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>you may be using --&gt; you might be using</p><p> </p><p>In the next sentence, remove "we". [Using second person is preferred](<a href="https://developers.google.com/style/person" target="_blank">https://developers.google.com/style/person</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>